src/components/Expander/ExpanderGroup.tsx

```diff
@@ -18,7 +18,7 @@ export const StyledPanel = styled((props: HtmlDivProps) => (
   max-width: 100%;
 `;
 
-interface PanelExpansionGroupState {
+interface ExpanderGroupState {
   /** Panels that are open */
   openPanels: number[];
```

Comment:
Some of these still speak of panels instead of expanders. The most obvious ones can be spotted from the styleguidist prop descriptions.